src/xokdinst.rs

```diff
@@ -201,6 +204,10 @@ struct GenConfigOpts {
     #[structopt(short = "V", long = "instversion")]
     /// Use a versioned installer binary
     installer_version: Option<String>,
+
+    /// Discover available cpu and memory resources for libvirt
+    #[structopt(long = "libvirt-auto-size")]
```

Comment:
This is fine as is but I think it'd also work to just say `#[structopt(long)]`.  Oh but I see we do this elsewhere.

Reply:
That works indeed.